homeassistant/components/imap_email_content/sensor.py

```diff
@@ -231,27 +232,43 @@ def get_msg_text(email_message):
         for part in email_message.walk():
             if part.get_content_type() == CONTENT_TYPE_TEXT_PLAIN:
                 if message_text is None:
+                    message_text_encoding = part["Content-Transfer-Encoding"]
```

Comment:
Is this header guaranteed to always exist?

Reply:
I have made change in code and added test to check there are no error if there is no "Content-Transfer-Encoding"